extensions/2.0/Vendor/EXT_mesh_features/schema/primitive.EXT_feature_metadata.schema.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
Make sure to update the filename of the JSON file, it still uses `EXT_feature_metadata`. There are a couple other files like this.